perf/load/values.yaml

```diff
@@ -1,4 +1,4 @@
-serviceGraphImage: tahler/isotope-service:1
+serviceGraphImage: gcr.io/istio-testing/isotope:0.0.1
```

Comment:
Who is responsible for building and pushing isotope image? Also the versioning.

Reply:
We could set up the full CI/CD but this image has changed 1 time in the past 1.5years+, so I am not sure its needed?